include/gazebo_mavlink_interface.h

```diff
@@ -200,6 +200,7 @@ class GazeboMavlinkInterface : public ModelPlugin {
 
 private:
   bool received_first_actuator_;
+  bool received_actuator_ = false;
```

Comment:
I'm not convinced that we need this one "global".

Reply:
Yes, you're right. Rolling back. I'm not sure myself why I did it :smile: